string label
geometry_msgs/PoseStamped pose
float32 probability
float32 colocationDist